### DISCUSSION TITLE
How to identify duplicate data between Chartevent and Labevent?

### DISCUSSION THREAD

**Discussion by Carl-Carl** [MIT-LCP/mimic-code#1214] [MIMIC-IV]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

I noticed that Chartevent and Labevent in MIMIC4 contain some duplicate data, according to the document. However, I am trying to remove these duplicate data for my research. I wonder if you could provide some info to help us identify these duplicate data so that the removing process will be easier. Thank you!

**Comment by alistairewj** [comment]
The easiest way is to look for data in *chartevents* where the `itemid` has a category related to laboratory data.

**Comment by Carl-Carl** [comment]
> The easiest way is to look for data in _chartevents_ where the `itemid` has a category related to laboratory data.

Thank you very much!

**Comment by pszolovits** [comment]
From my past experience, I would recommend using the data from labevents, not chartevents. IIRC, chartevents tends to contain a subset of the data in labevents, though there are a few more additional data items there.

**Comment by Carl-Carl** [comment]
> From my past experience, I would recommend using the data from labevents, not chartevents. IIRC, chartevents tends to contain a subset of the data in labevents, though there are a few more additional data items there.

I see. Thank you!
